species.rb

```diff
@@ -0,0 +1,12 @@
+require 'byebug'
+
+module Species
+extend self
+
+  def order_species species_data
+    # Critically Endangered, Endangered, Vulnerable
+    # CR is the most endangered, then comes EN, third VU
+    # Fortunately these are alphabetically ordered.s
+    species_data.sort_by! { |scientific_name, common_name, redlist_status, iucn_redlist_url| redlist_status }
```

Comment:
I think this could be simplified like:

```
species_data.sort_by! { |hash| hash['redlist_status'] }
```
The `species_data` variable should be an array of hashes so that should work.

As a side note, when you want to specify each element of an hash but you are not going to use all of those keys, you could use an underscore for simplicity, like:

```
species_data.sort_by! { |_, _, redlist_status, _| redlist_status }
```

Or at least use shorter variable names

Reply:
Thanks!